validate_rpm_specs.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+
+import argparse
+import glob
+import os
+import sys
+
+from lib import exception
+from lib.utils import is_package_installed
+from lib.utils import run_command
+
+def validate_rpm_spec(spec_file_path):
+
+    try:
+        run_command("rpmlint -v %s" % spec_file_path)
```

Comment:
Unrelated to this patch: we should probably have a file to keep those dev package requirements, like rpm_requirements.txt